content/departments/product-engineering/engineering/cloud/devops/index.md

```diff
@@ -27,28 +27,58 @@ This team is responsible for
 
 {{generator:product_team.devops}}
 
+## Contact & Support Guideline
+
+- The best way to contact the `cloud-devops` team is in the #cloud-devops slack channel. If it is urgent, please cc `@devops-support` in your message.
+- Issues with `team/devops` labels and [@team/cloud-devops](https://github.com/orgs/sourcegraph/teams/cloud-devops) team on GitHub
+- Please do not `@cloud-devops-team` or directly message individual teammates - this is to try and protect their focus
+
 ## Content
 
+- [Onboarding](onboarding.md)
 - [How to deploy a code change to the Cloud](deploy-code-change.md)
 - [Large release (rollout release) process](deploy-code-change.md#large-releases-to-cloud-rollup-releases)
 - [How to make configuration changes to sourcegraph.com](update_sg_website_config.md)
 - [Datadog monitoring](datadog.md)
-- [Onboarding](onboarding.md)
 - [How to add or modify DNS Records](dns.md)
 - [Disaster Recovery](disaster-recovery.md)
 - [How to resize disks in StatefulSet](resize-disks-in-statefulset.md)
 - [How to use preprod aka staging](preprod.md)
 - [Persistent disk backup schedule](persistent-disk-backup-schedule.md)
 - [Silencing Alerts](silencing-alerts.md)
 - [Cloud Postgres](cloud-postgres.md)
+- [Managed instances operation](./managed/index.md)
+
+## How we work
+
+### Issue tracking
+
+[The DevOps GitHub Project board](https://github.com/orgs/sourcegraph/projects/220/views/23) is the single source of truth.
+
+### Planning, Sync & Retro
 
-## How to contact the team and ask for help
```

Comment:
Maybe we should mention the slack support handle here?

Reply:
I moved it to the top for better visibility to other team https://github.com/sourcegraph/handbook/pull/3335/files#diff-1f6d0f991bbf3a06d1d5a82f14e8e408601bb1e5a9ce3f2ebd9ae6d84a571f94R30-R34

wdyt?

Reply:
Looks good!